### PR TITLE
Update banner for Q1 survey (to be launched Jan 30)

### PR DESCRIPTION
**Pull request by jayoung-lee**
I'd like to post a banner for Q1 survey, starting Jan 30th. It will stay open for a week (or a bit longer). Please make adjustments as needed!

[This is the previous PR](https://github.com/flutter/website/pull/1583)

### CODE REVIEW

File: src/_includes/banner.html [hunk 1]
```diff
@@ -6,5 +6,9 @@
       Flutter 1.0 has been released!
       <a href="{{fl_url}}" target="_blank" rel="noopener">Learn more</a>.
     </p>
+    <p>
+      Please take our 
+      <a href="https://google.qualtrics.com/jfe/form/SV_bfowrKmM2DjD5tj?Source=Website">survey</a> to help improve Flutter!
+    </p>
```
Comment (chalin) [inline src/_includes/banner.html]: How about:
> Help improve Flutter by taking our [survey](#)!

@sfshaza2 @kwalrath, WDYT?

Comment (chalin) [reply src/_includes/banner.html]: Or
> Help improve Flutter! Take our [survey](#)

Comment (sfshaza2) [reply src/_includes/banner.html]: I like your shorter suggestions, Patrice. I have no preference for A or B, though.  :)

**Comment by chalin** [comment]
> I like your shorter suggestions, Patrice. I have no preference for A or B, though.

Ok

> [Ja Young] Please make adjustments as needed!

Ok, I'll make the updates, thanks.

> [Ja Young] I'd like to post a banner for Q1 survey, starting Jan 30th.

So this can go live any time tomorrow (e.g., early in Eastern Time)?

**Comment by jayoung-lee** [comment]
@chalin Yes! Can we make it appear around 9 or 10am PST?

**Comment by chalin** [comment]
@jayoung-lee - ok for 9 or 10 am PST. Do you have any preferences for the suggested edits (options A or B, as Shams referred to them)?

**Comment by chalin** [comment]
Here is a sample screenshot:
screen shot 2019-01-29 at 13 53 23

**Comment by jayoung-lee** [comment]
Option B sounds good. Could you just add a period at the end, just like the 1.0 announcement?:

"Help improve Flutter! 
Take our survey."

**Comment by chalin** [comment]
> add a period

Done

**Comment by jayoung-lee** [comment]
Perfect. Thank you 😃

**Comment by chalin** [comment]
The banner is live.

**Pull request merged**